examples/javascript/button.js

```diff
@@ -26,7 +26,7 @@
 var groveSensor = require('jsupm_grove');
```

Comment:
Ah, I just realised that there are separate examples for [`button.js`](https://github.com/intel-iot-devkit/upm/blob/master/examples/javascript/button.js) and [`grovebutton.js`](https://github.com/intel-iot-devkit/upm/blob/master/examples/javascript/grovebutton.js). I see what you mean now.

@Propanu Sure, I'll open an issue.